code/android-core-library/build.gradle

```diff
@@ -148,7 +152,7 @@ publishing {
     }
 }
 tasks.withType(Javadoc).all { enabled = false }
-publish.dependsOn('assemblePhone')
+publishToMavenLocal.dependsOn('assemblePhone')
```

Comment:
Can we keep both the lines?

Reply:
I will update it in a future PR to publish the release build.